homeassistant/components/light/services.yaml

```diff
@@ -2,28 +2,44 @@
 
 turn_on:
   description: Turn a light on.
+  target:
+    name: Target
+    description: What to turn on
+    example: "light.kitchen"
```

Comment:
Let's drop example. It's also not correct, it should be `entity_id: light.kitchen`.